plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -292,4 +298,28 @@ private URL getIncrementalsMirror() {
         }
         return jenkinsIncrementalsRepo;
     }
+
+    /**
+     * Prints out the Plugin Management Tool version
+     */
+    public void showVersion() {
+        try (InputStream propertyInputStream = this.getClass().getResourceAsStream("/.properties")) {
+            if (propertyInputStream == null) {
+                System.out.println("Unable to find .properties");
```

Comment:
This sounds like an error case,
how about?
```suggestion
                System.out.println("No version information available");
                System.exit(1);
```